xlml/utils/metric.py

```diff
@@ -266,6 +266,8 @@ def process_tensorboard_summary(
 
   if summary_config.use_regex_file_location:
     file_location = get_gcs_file_location_with_regex(file_location)
+    if file_location == "":
+      return [[]], [[]]
```

Comment:
Can you double check if we return `[], []`? it seems this method return 2 lists.

Reply:
I'm seeing this method returns (List[List[]], List[List[]]) https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/83da803d0aa587cf1c6f6b80ddc0aa39fa601f7f/xlml/utils/metric.py#L234-L242